src/content/docs/ai-monitoring/explore-ai-data/view-model-data.mdx

```diff
@@ -0,0 +1,167 @@
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Performance</DoNotTranslate>**: View data about your AI model's performance.
+</figcaption>
+
+The performance tab aggregates response and request metrics across all your models. Overview the models that take the most time to process a request or create a response with the pie charts, or refer to the time series graphs to track upticks in request or response times. You can use performance charts to locate outliers across your models. 
+
+### Cost tab [#cost-inventory]
+
+<img
+    title="Model inventory: Performance"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryCostPage}
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Cost</DoNotTranslate>**: View data about your AI model's cost.
+</figcaption>
+
+The cost tab uses a combination of time series graphs and pie charts to identify cost drivers amongst your models. Determine the number of tokens came from either prompts or completions, or if certain models cost more on average than others.  
+
+* <DoNotTranslate>**Tokens used and token limit**</DoNotTranslate>: Evaluate how often your models approach a given token limit.
+* <DoNotTranslate>**Total tokens by models**</DoNotTranslate>: Determine which of your models use the most tokens on average.
+* <DoNotTranslate>**Total usage by prompt and completion tokens**</DoNotTranslate>: Understand what ratio of tokens comes from prompts your model accepts against tokens used per completion. 
+
+Understanding cost allows you to improve how your AI app uses one or more of your models so your AI toolchain is more cost effective. 
+
+## Model comparison page [#model-comparison]
+
+<img
+    title="Model inventory overview"
+    alt="A screenshot of the overview page when you go to Model inventory"
+    src={aiModelComparisonPage}
+/>
+
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model comparison</DoNotTranslate>**: Compare data about the different AI models in your stack.
+</figcaption>
+
+The model comparison page organizes your AI monitoring data to help you conduct comparative analysis. This page scopes your model comparison data to a single account, giving you aggregated data about model cost and performance across one or more apps. To generate data:
+
+1. Choose your models from the drop down.
+1. Scope to one service to see performance in the context of a particular app, or keep the query to `Service = All` to see how a model behaves on average. 
+1. Choose time parameters. This tool is flexible: you can make comparisons across different time periods, which lets you see how performance or cost changed before and after a deployment.  
+
+### Compare model performance [#compare-performance]
```

Comment:
What do you think of making the compare model performance and compare model cost sections collapsers?